src/logic/safe/store/actions/processTransaction.ts

```diff
@@ -57,7 +56,7 @@ interface ProcessTransactionArgs {
     refundReceiver: string
   }
   userAddress: string
-  ethParameters?: Pick<TxParameters, 'ethNonce' | 'ethGasLimit' | 'ethGasPriceInGWei'>
+  ethParameters?: Pick<TxParameters, 'ethNonce' | 'ethGasLimit' | 'ethGasPriceInGWei' | 'ethMaxPrioFeeInGWei'>
```

Comment:
I don't fully remember how we achieve this in the refactor, but this is a duplicate of the type from the creation function. Maybe we can export it and use it here as well?

Reply:
Out of scope for this PR.

Reply:
The extra parameters are added here though, not in the other.

Reply:
Yes, but the duplicate argument types are better addressed in the create/process refactoring, not here.

Reply:
I guess that'll have to be included in the 'v2' refactor then because the new parameters add here are not there.